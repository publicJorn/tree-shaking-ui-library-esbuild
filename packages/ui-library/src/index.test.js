import * as designSystem from './index'

test('exposed design system parts', () => {
  const keys = Object.keys(designSystem)
  expect(keys).toMatchSnapshot()
})
