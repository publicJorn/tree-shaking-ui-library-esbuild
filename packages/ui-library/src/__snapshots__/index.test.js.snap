// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exposed design system parts 1`] = `
Array [
  "default",
  "Button",
  "GlobalStyles",
  "Link",
  "darkTheme",
  "defaultTheme",
]
`;
